sa/model.go

```diff
@@ -12,23 +12,137 @@ import (
 	"github.com/letsencrypt/boulder/core"
 	"github.com/letsencrypt/boulder/features"
 	"github.com/letsencrypt/boulder/probs"
+	"github.com/letsencrypt/boulder/revocation"
 )
 
-const (
-	regV1Fields        string = "id, jwk, jwk_sha256, contact, agreement, initialIP, createdAt, LockCol"
-	regV2Fields        string = regV1Fields + ", status"
-	pendingAuthzFields string = "id, identifier, registrationID, status, expires, combinations, LockCol"
-	authzFields        string = "id, identifier, registrationID, status, expires, combinations"
-	sctFields          string = "id, sctVersion, logID, timestamp, extensions, signature, certificateSerial, LockCol"
+type dbSelectOne func(interface{}, string, ...interface{}) error
+type dbSelect func(interface{}, string, ...interface{}) ([]interface{}, error)
 
-	// CertificateFields and CertificateStatusFields are also used by cert-checker and ocsp-updater
-	CertificateFields       string = "registrationID, serial, digest, der, issued, expires"
-	CertificateStatusFields string = "serial, subscriberApproved, status, ocspLastUpdated, revokedDate, revokedReason, lastExpirationNagSent, ocspResponse, LockCol"
+const regFields = "id, jwk, jwk_sha256, contact, agreement, initialIP, createdAt, LockCol"
```

Comment:
I think these consts are only used in one place and can be put directly into the query string, right?